src/pages/GamePage/Gameplay/CarFactory.js

```diff
@@ -3,89 +3,90 @@ import { collisionCategories } from "./Constants";
+  //   LFTire.setCollidesWith(tireWorldCollision);
 
-//   RFTire.setCollisionCategory(collisionCategories["TIRE"]);
-//   RFTire.setCollidesWith(tireWorldCollision);
+  //   RFTire.setCollisionCategory(collisionCategories["TIRE"]);
+  //   RFTire.setCollidesWith(tireWorldCollision);
 
-//   LRTire.setCollisionCategory(collisionCategories["TIRE"]);
-//   LRTire.setCollidesWith(tireWorldCollision);
+  //   LRTire.setCollisionCategory(collisionCategories["TIRE"]);
+  //   LRTire.setCollidesWith(tireWorldCollision);
 
-//   RRTire.setCollisionCategory(collisionCategories["TIRE"]);
-//   RRTire.setCollidesWith(tireWorldCollision);
+  //   RRTire.setCollisionCategory(collisionCategories["TIRE"]);
+  //   RRTire.setCollidesWith(tireWorldCollision);
 
   var Bodies = Phaser.Physics.Matter.Matter.Bodies;
   var carPhysicsBody = Bodies.rectangle(0, -600, 100, 62, {
     mass: 1000,
-    inverseMass: 1/1000,
+    inverseMass: 1 / 1000,
     friction: 0.3,
-    fritctionAir: 0.4
+    fritctionAir: 0.4,
+    label: "Car body"
   });
 
   var carBody = phaser.matter.add.image(0, 0, "wheel");
   carBody.displayWidth = 100;
   carBody.displayHeight = 62;
-  
+
   carBody.setExistingBody(carPhysicsBody);
-//   carBody.setCollisionCategory(collisionCategories["BODY"]);
-//   carBody.setCollidesWith([
-//     collisionCategories["WORLD"],
-//     collisionCategories["BODY"]
-//   ]);
-
-//   var LFTireAttach = createAttachPoint(
-//     LFTire.body,
-//     carBody.body,
-//     Phaser.Physics.Matter.Matter.Vector.create(0, 0),
-//     Phaser.Physics.Matter.Matter.Vector.create(-50, 65)
-//   );
-//   phaser.matter.world.add(LFTireAttach);
-
-//   var RFTireAttach = createAttachPoint(
-//     RFTire.body,
-//     carBody.body,
-//     Phaser.Physics.Matter.Matter.Vector.create(0, 0),
-//     Phaser.Physics.Matter.Matter.Vector.create(50, 65)
-//   );
-//   phaser.matter.world.add(RFTireAttach);
-
-//   var LRTireAttach = createAttachPoint(
-//     LRTire.body,
-//     carBody.body,
-//     Phaser.Physics.Matter.Matter.Vector.create(0, 0),
-//     Phaser.Physics.Matter.Matter.Vector.create(-50, -65)
-//   );
-//   phaser.matter.world.add(LRTireAttach);
-
-//   var RRTireAttach = createAttachPoint(
-//     RRTire.body,
-//     carBody.body,
-//     Phaser.Physics.Matter.Matter.Vector.create(0, 0),
-//     Phaser.Physics.Matter.Matter.Vector.create(50, -65)
-//   );
-//   phaser.matter.world.add(RRTireAttach);
-
-//   var carComposite = Phaser.Physics.Matter.Matter.Composite.create({
-//     bodies: [LFTire.body, RFTire.body, LRTire.body, RRTire.body, carBody.body],
-//     lineColor: "FFFFFF"
-//   });
-
-//   phaser.matter.world.add(carComposite);
+  //   carBody.setCollisionCategory(collisionCategories["BODY"]);
+  //   carBody.setCollidesWith([
+  //     collisionCategories["WORLD"],
+  //     collisionCategories["BODY"]
+  //   ]);
+
+  //   var LFTireAttach = createAttachPoint(
+  //     LFTire.body,
+  //     carBody.body,
```

Comment:
Number of lines with comments kinda confuse me whether they are significant or not - I suggest not commiting such wall of text

Reply:
As said in conversation with Olaf - it probably will be useful in the future so I've left this here.

Reply:
Understood